features/facebookImport/src/facebookImporter.jsx

```diff
@@ -113,6 +113,7 @@ const FacebookImporterApp = () => {
     return (
         <Router history={history}>
             <ImporterProvider>
+                <div className="poly-nav-bar-separator-overlay" />
```

Comment:
Seemed like the nicest place to add it, since we don't use the routing logic for our little loading screen in `FacebookImporter` yet. But open for suggestions :D

Reply:
Could maybe also refactor out a react feature component that also makes a router and context and then put it there. For now seems good enough for sure to me